.github/workflows/ci.yml

```
@@ -238,28 +238,36 @@ jobs:
           checks/requirements.py
 
   notebook:
-    name: Notebook check
     env:
-      SUPERSTAQ_API_KEY : ${{ secrets.SUPERSTAQ_API_KEY }}
-      IBMQ_TOKEN : ${{ secrets.IBMQ_TOKEN }}
+      SUPERSTAQ_API_KEY: ${{ secrets.SUPERSTAQ_API_KEY }}
+      IBMQ_TOKEN: ${{ secrets.IBMQ_TOKEN }}
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - type: "qiskit-superstaq"
+            pattern: "*_qss.ipynb qiskit-superstaq"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples]"
+          - type: "cirq-superstaq"
+            pattern: "*_css.ipynb cirq-superstaq"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./cirq-superstaq[examples]"
+          - type: "other docs"
+            pattern: "--exclude *_css.ipynb *_qss.ipynb 'cirq-superstaq/*' 'qiskit-superstaq/*'"
+            dependencies: "-e ./checks-superstaq -e ./general-superstaq[dev] -e ./qiskit-superstaq[dev] -e ./cirq-superstaq[dev] -e ./supermarq-benchmarks[dev]"
```

(sorry one more nit)
```suggestion
            dependencies: "-e ./checks-superstaq -e ./general-superstaq -e ./qiskit-superstaq[examples] -e ./cirq-superstaq[examples] -e ./supermarq-benchmarks"
```